src/java/org/jnativehook/GlobalScreen.java

```diff
@@ -50,7 +51,13 @@
 	/**
 	 * Logging service for the native library.
 	 */
-	protected static Logger log = Logger.getLogger(GlobalScreen.class.getPackage().getName());
+	protected static Logger logger = Logger.getLogger(GlobalScreen.class.getPackage().getName());
+	static {
```

Comment:
so my big concern with this is will it handle the standard Java LogManager properties?  See https://docs.oracle.com/cd/E17277_02/html/GettingStartedGuide/managelogging.html#logginglevels

Most of the people complaining about this have no idea how Java's logging facility works.

Reply:
Because it isn't good. I have already mentioned slf4j, but it is your choice whether you want to use it. Otherwise, there should be a way of checking these properties, but as long as #250 is unresolved I can hardly test it.